### ISSUE TITLE
When project names change...stuff starts breaking

### ISSUE THREAD

**Issue by TylerRadford** [hotosm/hotosm-website#280]
![image](https://user-images.githubusercontent.com/9853903/41182990-2ca7618e-6b46-11e8-80d0-d1c9b95ac41e.png)

For example, the "What We Do" page links to specific projects. It seems to link based on the project name -- when these change, the link breaks. In the screenshot above, the project listed no longer exists by that name.

**Comment by smit1678** [comment]
Yeah, this isn't 100% stable. @brendangatens what do you think our options are here to add some integrity of the linking?
